MediaBrowser.Controller/Plugins/ILongRunningTask.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using System.Threading.Tasks;
+
+namespace MediaBrowser.Controller.Plugins
+{
+    /// <summary>
+    /// Interface ILongRunningTask
```

Comment:
This should be properly documented to explain how the lifetime is managed by application host and the difference between `IEntryPoint`